packages/edit-post/src/components/sidebar/settings-sidebar/index.js

```diff
@@ -30,17 +31,22 @@ const SettingsSidebar = ( { sidebarName } ) => (
 		<SettingsHeader sidebarName={ sidebarName } />
 		<Panel>
 			{ sidebarName === 'edit-post/document' && (
-				<Fragment>
-					<PostStatus />
-					<LastRevision />
-					<PostLink />
-					<PostTaxonomies />
-					<FeaturedImage />
-					<PostExcerpt />
-					<DiscussionPanel />
-					<PageAttributes />
-					<MetaBoxes location="side" />
-				</Fragment>
+				<PluginDocumentSetting.Slot>
+					{ ( fills ) => (
+						<Fragment>
+							<PostStatus />
+							{ fills }
```

Comment:
Out of curiosity, why do fills are placed after `<PostStatus />`? Why not before `<MetaBoxes location="side" />` or any other place?

Reply:
The placement was chosen based on a pretty common request ( at least where I work ) and it seemed appropriate to place it below PostStatus as most items being added would be of less importance than the data displayed there.

In a perfect world, being able to insert items between all items would be ideal but this is a good start.